src/bin/pg_dump/dump_babel_utils.c

```diff
@@ -121,8 +121,9 @@ static bool
 isBabelfishConfigTable(TableInfo *tbinfo)
 {
 	/* We don't want to dump logins during logical database dump */
-	if (tbinfo && tbinfo->dobj.namespace &&
-		strcmp(tbinfo->dobj.name, "babelfish_authid_login_ext") == 0)
+	if (tbinfo->relkind != RELKIND_RELATION ||
```

Comment:
if we are checking for tbinfo being not null, it should come first

Reply:
Fixed!